src/main/java/taxi/controller/login/LogoutController.java

```diff
@@ -0,0 +1,16 @@
+package taxi.controller.login;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getSession().invalidate();
+        resp.sendRedirect("/logins/login");
```

Comment:
Another place, but the same mistake. You should check all your PR before requesting a review.
![image](https://user-images.githubusercontent.com/20326858/203294117-8d1c4ca3-349c-4631-95c8-e595450d5b68.png)